torch/_dynamo/variables/flash_attention.py

```diff
@@ -0,0 +1,26 @@
+from ..bytecode_transformation import create_call_function
+from .base import VariableTracker
+
+
+class SDPAParamsVariable(VariableTracker):
+    """Represents the c++ params struct for flash attention"""
+
+    def __init__(self, proxy, param_vars, **kwargs):
+        self.proxy = proxy
+        self.param_vars = param_vars
+        super().__init__(**kwargs)
+
+    def reconstruct(self, codegen):
+        codegen.load_import_from("torch._C", "_SDPAParams")
+        for var in self.param_vars:
+            codegen(var)
+        return create_call_function(len(self.param_vars), True)
```

Comment:
Are you sure you always want to make a fresh one here? What if it exists as a global? proxy objects should be tracked as proper first class citizens in codegen, check how tensros work.

Reply:
Hmm, if it has a source, won't codegen use the source instead and not call reconstruct?